### ISSUE TITLE
Handle images

### ISSUE THREAD

**Issue by vi-cat** [SnowdogApps/wombat#16]
How does Wombat expect images to be used?

For example, if a `content.md` had:
```
![](image.png)
```

Could it be possible for the endpoint to also return the image?

**Comment by Igloczek** [comment]
Nah, it's up to you to handle URLs to images, we internally always did it inside the frontend app.

Wombat just serves JSON responses, since it doesn't know where you will deploy the lambda and the lambda itself is stateless, it doesn't know its own URL.
